lib/config-loader.js

```diff
@@ -3,6 +3,7 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+var cloneDeep = require('lodash').cloneDeep;
```

Comment:
lodash has a lot of useful utility functions, might make more sense to require it on its own so that someone can use other lodash functions without changing this line.